### DISCUSSION TITLE
Tag a new release

### DISCUSSION THREAD

**Discussion by tbeason** [duckdb/duckdb#8818] [General]
I think the pace of development warrants a faster release cadence. Multiple times I've seen people complain about issues on 0.8.1 that have been fixed for months on master branch.

**Comment by carlopi** [comment]
Tentative: https://duckdb.org/dev/release-dates

**Comment by szarnyasg** [comment]
As the link by Carlo shows, the new release is scheduled on Sep 18. We would like to ensure that our releases are stable and well-tested, hence we have a one-week feature freeze before each release. It is unlikely that we'll change our release cadence in the near future.
